whetstone/src/main/java/com/deliveryhero/whetstone/Whetstone.kt

```diff
@@ -136,13 +142,17 @@ public object Whetstone {
      * @see [ContributesServiceInjector]
      */
     public fun inject(service: Service) {
+        GlobalAndroidComponentListener.componentInjectionListener
+            ?.onInjectStart(InjectedComponent.InjectedService(service))
         val app = service.application
         val injector = fromApplication<ServiceComponent.ParentComponent>(app)
             .getServiceComponentFactory()
             .create(service)
             .membersInjectorMap[service.javaClass] as? MembersInjector<Service>
 
         requireNotNull(injector).injectMembers(service)
+        GlobalAndroidComponentListener.componentInjectionListener
+            ?.onInjectFinish(InjectedComponent.InjectedService(service))
```

Comment:
what about other injected components? app and views

Reply:
It was defined in the task for these two but I'm ok to add for those two as well.

Reply:
I added those two in another commit. I'm fan of having those two as well 👍